styles.css

```diff
@@ -0,0 +1,104 @@
+* {
+  position: relative;
+  font-family: sans-serif, Poppins;
+  z-index: 2;
+}
+
+.logo {
+  margin-right: 0;
+  padding-left: 50%;
+  width: 100%;
+  color: #6070ff;
+  font-size: 18px;
+}
+
+.top {
+  width: 100%;
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  top: 4px;
+  z-index: 3;
+}
+
+.menuhover {
+  padding: 7px;
+  margin-right: 30px;
+  margin-left: 10px;
+  margin-bottom: 10px;
+  margin-top: 13px;
+  width: 15%;
+}
+
+.menuhover button {
+  border: none;
+  background-color: inherit;
+  color: #6070ff;
+  font-size: 24px;
+}
+
+.intro {
+  width: 100%;
+}
+
+.text {
+  padding-top: 100px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 130%;
+  padding-right: 24px;
+  padding-left: 24px;
+  padding-bottom: 12px;
+  color: #172b4d;
+  margin-bottom: 0;
+  z-index: 2;
+}
+
+.text2 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 150%;
+  padding-right: 24px;
+  padding-left: 24px;
+  color: #344563;
+  width: 90%;
+  margin-top: 0;
+  margin-bottom: 12px;
+}
```

Comment:
### Required changes
- Your font-size and line-height do not currently replicate the Figma design. Kindly refer to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=aMdrZuZl7iUp36JB-0) for details

Reply:
font-family: 'Poppins';
font-style: normal;
font-weight: 400;
font-size: 16px;
line-height: 24px;
/* or 150% */

this is the info from figma, it does match 
 
requirments said to use porcentage instead of a px, that why i put line heigth on 150% as figma design info told to

Reply:
@Eternalgratis

Reply:
@Eternalgratis hey, i didnt add screen size since there was no requirements for changins the top menu, so i did everything responsive, it will work for mobile or desktop. what  changes should i make there

Reply:
@Eternalgratis im waiting for your reply so i can put another review request, thanks